build/rbac-setup.sh

```diff
@@ -11,7 +11,6 @@
 # DIR="$( find $HOME -type d -name "search-e2e-test" )"
 # using abosolute path, relative path is not working in travis
 
-set -e
```

Comment:
Add an `echo` message to remind us that this is disabled.

Reply:
Readding this error flag.